cfme/infrastructure/provider/kubevirt.py

```diff
@@ -4,6 +4,9 @@
 
 
 class KubeVirtProvider(InfraProvider):
+    type_name = "kubevirt"
+    settings_key = 'ems_kubevirt'
+    mgmt_class = None
```

Comment:
Do eventually want wrapanapi to support kubevirt?

Reply:
I believe so, the class doesn't exist yet though.